src/extent.c

```diff
@@ -1319,21 +1319,17 @@ extent_grow_retained(tsdn_t *tsdn, arena_t *arena,
 		 * cant_alloc case should not occur.
 		 */
 		assert(result == extent_split_interior_error);
+		if (to_salvage != NULL) {
+			extent_deregister(tsdn, to_salvage);
```

Comment:
Should we do extent_deregister_no_gdump_sub here as well?

Reply:
I don't think so; whatever we do to the salvageable extent counted as a "real" extent operation (unlike the leak path).

Reply:
Hmm however we added the extent with `extent_register_no_gdump_add(tsdn, extent)` L1282 this file?